src/test/libcephfs/test.cc

```diff
@@ -1374,3 +1374,42 @@ TEST(LibCephFS, OpenNoClose) {
   // shutdown should force close opened file/dir
   ceph_shutdown(cmount);
 }
+
+TEST(LibCephFS, TestStatx) {
+  struct ceph_mount_info *cmount;
+  ASSERT_EQ(ceph_create(&cmount, NULL), 0);
+  ASSERT_EQ(ceph_conf_read_file(cmount, NULL), 0);
+  ASSERT_EQ(0, ceph_conf_parse_env(cmount, NULL));
+  ASSERT_EQ(ceph_mount(cmount, "/"), 0);
+  int64_t old_btime;
+  int32_t old_btime_ns;
+
+
+  char filename[32];
+  sprintf(filename, "/getattrx%x", getpid());
+
+  /* FIXME: check for errors here */
```

Comment:
Yeah, though TBH we don't really care whether that succeeds or fails. Either it succeeds and the file is removed, or it doesn't and the file isn't. What really matters is whether the O_CREAT open afterward succeeds. We can probably just remove that comment.